POS/Integration/06a_CreateStoreManually.md

```diff
@@ -0,0 +1,500 @@
+
+#### Prerequisites
+
+A store has been created, see [Create a store manually](#create-a-store-manually) or [Create a store using the wizard](./06_CreateStore.md#create-a-store-using-the-wizard).
+
+#### Procedure
+
+*Venduo POS > Management > Tab STORES > Select Store > Tab Settings*
+
+![Store Basic Data](../../Assets/Screenshots/POS/Management/Stores/Store/Settings/Settings.png "[Store Basic Data]")
+
+1. Select the *Store address* entry in the list of settings in the left column.   
+    The *Store address* view is displayed on the right side.
+
+    ![Store Basic Data](../../Assets/Screenshots/POS/Management/Stores/Store/Settings/StoreAddress.png "[Store Basic Data]")
+
+2. Enter the store address in the fields.
+
+    > [Info] The store address corresponds to the delivery address. You must at least enter the country and the ZIP code.   
+
+    For detailed information, see [Stores](../UserInterface/02b_Stores.md).
+
+3. Click the [Save] button in the upper right corner.   
+    A confirmation message is displayed. The store address is saved.
+
+
+## Assign the POS warehouse
+
+To connect the warehouse management and the stock management to your POS, you have to assign a warehouse from which to take the materials. The warehouse specifies where exactly the material is taken from when a product is sold.
+
+#### Prerequisites
+
+- A store has been created, see [Create a store manually](#create-a-store-manually) or [Create a store using the wizard](./06_CreateStore.md#create-a-store-using-the-wizard).
+- A warehouse for POS is configured, see [Configure the warehouse for POS](./01_ConfigureWarehouse.md).
+
+#### Procedure
+
+*Venduo POS > Management > Tab STORES > Select Store > Tab Admin Settings*
+
+![Store Admin Settings](../../Assets/Screenshots/POS/Management/Stores/Store/AdminSettings/AdminSettings.png "[Store Admin Settings]")
+
+1. Select the *Warehouse assignment orders* entry in the list of settings in the left column.   
+    The *Warehouse assignment orders* section is displayed on the right side.
+
+    ![Warehouse assignment orders](../../Assets/Screenshots/POS/Management/Stores/Store/AdminSettings/WarehouseAssignment.png "[Warehouse assignment orders]")
+
+2. Select the warehouse you want to take the material from in the *Warehouse* drop-down list.
+
+3. Activate the *Automatically assign product to warehouse when activated in store* toggle to automate the assignment process. It is recommended to activate this option.   
+For detailed information, see [Stores](../UserInterface/02b_Stores.md).
+
+   > [Info] A product must have stock allocation in a warehouse in order to be sold from that warehouse. Products only have a stock in a warehouse if a shelf is defined for them. By default, a shelf is not defined for products. Therefore, each product that is to be sold in the POS system must first get a shelf. The products get a shelf the first time the corresponding channels offer is activated.    
```

Comment:
Vgl. mit dem Info in Create store using wizard. Es gibt Abweichungen zwischen den beiden. Z.B.: A product must have "stock allocation"? Ich verstehe nicht ganz, was damit gemeint ist. Vielleicht eher "a product must be in stock in a warehouse..."? 

Zum Anlege des Fachs: Wenn man im manuellen Anlagen eines Stores auch automatisch den Storage shelf anlegen kann, wie es hier aussieht, braucht man dann in Integration keine Storage shelves anzulegen, offenbar. Also mein Kommentar oben kann man ignorieren.